src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -42,7 +42,7 @@ class TierCalculationService(
   @Transactional
   fun calculateTierForCrn(crn: String) =
     calculateTier(crn).let {
-      val isUpdated = it.data != getLatestTierCalculation(crn)?.data
+      val isUpdated = isUpdated(it, crn)
```

Comment:
this was fine before we added the breakdown of individual scores which has changed the equals().